tasks/main-blivet.yml

```diff
@@ -138,10 +138,9 @@
 # Manage /etc/crypttab
 #
 - name: Manage /etc/crypttab to account for changes we just made
-  crypttab:
-    name: "{{ entry.name }}"
-    backing_device: "{{ entry.backing_device }}"
-    password: "{{ entry.password }}"
+  lineinfile:
```

Comment:
Hi @dwlehman, do you know if we can use the `cryptsetup` command for this? I didn't find a way to set password via it, and the `crypttab` [module](https://gitlab.ics.muni.cz/242618/ansible/-/blob/6a10514e086b839604d819cca8c2d19dd0362923/lib/ansible/modules/system/crypttab.py) simply handles the config file, so I ended up using `lineinfile`, but maybe you can help.
Thank you

Reply:
Are the changes I made good to go then?

Reply:
> Are the changes I made good to go then?

I think so - let's see if we can get some better test results.

[citest]